skytap/resource_skytap_vm.go

```diff
@@ -468,7 +501,7 @@ func resourceSkytapVMUpdate(d *schema.ResourceData, meta interface{}) error {
 
 	opts := skytap.UpdateVMRequest{}
 
-	if v, ok := d.GetOk("name"); ok {
+	if v, ok := d.GetOk("name"); ok && d.HasChange("name") {
```

Comment:
This HasChange here is a bit odd

Reply:
... and duplicated on the next line too. Will handle under #18

Reply:
I left the HasChange because there isn't a way AFAIK to remove a name once you have set it.